.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
For editor-specific settings, it is recommended to add to a global gitignore file (~/.config/git/ignore).

* https://github.com/crossbeam-rs/crossbeam/blob/master/.gitignore
* https://github.com/tokio-rs/tokio/blob/master/.gitignore